libafl/src/corpus/mod.rs

```diff
@@ -25,8 +25,10 @@ pub use cached::CachedOnDiskCorpus;
 pub mod minimizer;
 use core::{cell::RefCell, fmt};
 
+pub mod null;
```

Comment:
We mostly named similar things `nop` iirc

Reply:
Just double checked, there is a `NopMonitor`, for example - so I'd suggest renaming this to `NopCorpus` :)

Reply:
Done.